### PR TITLE
Portofolio mobileversion

### PR DESCRIPTION
**Pull request by Vianney992**
we build the middle section of the mobile website:
- the work section
- the about myself section

### CODE REVIEW

File: project1.css [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 418px;
+  left: calc(50% - 375px / 2);
+  top: 552px;
+}
```
Comment (mukhammadrizooff) [inline project1.css:40]: - [ ] [REQUIRED] Good job so far! The design in `Works` and `About myself` section of your project is not exactly like [Figma design](https://www.figma.com/files/recent?fuid=1045426358203651467), please correct them.

image

File: project1.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+<button type="button">html</button>
+<button type="button">css</button>
+<button type="button">javaScript</button>
+
+
+<button type="button">See Project</button>
+    
+</section>
+-->
```
Comment (mukhammadrizooff) [inline project1.html]: - [ ] [OPTIONAL] Please clean these comments from your code base.

File: project1.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+<div class="four4">
+<div><button>html</button></div>
+<div></div>
+<div></div>
+<div><button>css</button></div>
+<div></div>
+<div></div>
+<div><button>JavaScript</button></div>
+</div>
```
Comment (mukhammadrizooff) [inline project1.html]: - [ ] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.

File: project1.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+  <div class="flex-container5">
+    <div><a><img src="Ellipse 1 (2).jpg" alt="CSS"></a></div>
+    <div><h5>CSS</h5></div>
+    <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  </div>
```
Comment (mukhammadrizooff) [inline project1.html]: - [ ] [REQUIRED] Please make changes! The technology in each category (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.

File: project1.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+  <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a></div>
+
+  <div><h4 class="Skills">Skills</h4></div>
+
+    <a class="Separator2"><img src="Separator Bottom.jpg" alt=""></a>
+  
+    <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a>
+
+</div>
```
Comment (mukhammadrizooff) [inline project1.html]: - [ ] [REQUIRED] Please make changes! The technology categories ("Languages", "Frameworks" and "Skills") should be a list `ul` and each of them a `li`.

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Vianney992 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Descriptive commit history ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: project1.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+      <button class="button2">css</button>
+      <button class="button3">javascript</button>
+    </li>
+  </ul>
+
+</div>
+
+<div><button class="button4">See Project</button></div>
+
+</div>
```
Comment (mwanawabangona) [inline project1.html]: - [ ] `optional`
- [ ] please add hover or a cursor pointer so the user can know that its a button

File: project1.html [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+   <div class="one">LET'S CONNECT </div>
+
+      <a class="github"><img src="github.jpg" alt=" "></a>
+      <a class="linkedin"><img src="Linkedinicon.jpg" alt=" "></a>
+      <a class="angellist"><img src="angellist.jpg" alt=" "></a>
+      <a class="twitter"><img src="twitter.jpg" alt=" "></a>
+      <a class="medium"><img src="medium.jpg" alt=" "></a>
+
+      <div><button class="two">Get my resume</button></div>
+</div>
```
Comment (mwanawabangona) [inline project1.html]: - [ ] `optional`
- [ ] please add hover or a cursor pointer so the user can know that its a button

File: project1.html [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+    Look through some of my work and experience! If you like what you see and have
+    a project you need coded, don't hesitate to contact me.</p></div>
+
+   <div class="one">LET'S CONNECT </div>
+
+      <a class="github"><img src="github.jpg" alt=" "></a>
+      <a class="linkedin"><img src="Linkedinicon.jpg" alt=" "></a>
+      <a class="angellist"><img src="angellist.jpg" alt=" "></a>
+      <a class="twitter"><img src="twitter.jpg" alt=" "></a>
+      <a class="medium"><img src="medium.jpg" alt=" "></a>
```
Comment (mwanawabangona) [inline project1.html]: - [ ] `required`
- [ ] please  add the list of icons using a `ul` tag to group them, and `li` per icon; each icon has a link (a tag) like the previous reviewer requested. it should look something like the screenshot below
Screenshot from 2022-06-29 13-54-35

File: project1.html [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+  <div class="flex-container3">
+  <div><a><img src="Ellipse 1.jpg" alt="Javascript"></a></div>
+  <div><h5> Javascript</h5></div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  </div>
```
Comment (mwanawabangona) [inline project1.html:88]: - [ ] `optional`
- [ ] please clean up the code you are not using in order for your code to look clean and organized

File: project1.html [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+  <div class="flex-container4">
+  <div><a><img src="Ellipse 1 (1).jpg" alt="HTML"></a></div>
+  <div><h5>HTML</h5></div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
```
Comment (mwanawabangona) [inline project1.html:96]: - [ ] `optional`
- [ ] please clean up the code you are not using in order for your code to look clean and organized

File: project1.html [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+  </div>
+
+  <div class="flex-container5">
+    <div><a><img src="Ellipse 1 (2).jpg" alt="CSS"></a></div>
+    <div><h5>CSS</h5></div>
+    <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
```
Comment (mwanawabangona) [inline project1.html:105]: - [ ] `optional`
- [ ] please clean up the code you are not using in order for your code to look clean and organized

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @Vianney992 

###  You have done an amazing job so far!  good job implementing some of the changes requested by the previous reviewer👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! pinching_hand


## Highlights
* ✅  No linter errors heavy_check_mark
* ✅  Repo looks professional briefcase
* ✅  Solid pull request scroll
* ✅  Good use of flexbox 🧐
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: project1.html (hunk 8 above)
Comment (Vianney992) [reply project1.html]: @mwanawabangona  adding a list degrade the style (enumerate buttons appears).

File: project1.html [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+
+   <div class="one">LET'S CONNECT </div>
+
+<div>
+     <a ><img src="github.jpg" alt=" " class="github"></a>
+    <a ></a><img src="Linkedinicon.jpg" alt=" " class="linkedin"></a>
+      <a ><img src="angellist.jpg" alt=" " class="angellist"></a>
+      <a ><img src="twitter.jpg" alt=" " class="twitter"></a>
+      <a ><img src="medium.jpg" alt=" " class="medium"></a>
+</div>
```
Comment (roshan-bajgain) [inline project1.html]: - [ ]  please add the list of icons using a ul tag to group them and li per icon; each icon has a link (a tag) as requested by the previous reviewer. This is one of the best ways to list the items. Kindly implement and we need to stick to the comment said by the previous reviewer

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Vianney992, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: project1.html [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+  <ul>
+     <li><a ><img src="github.jpg" alt=" " class="github"></a></li>
+    <li><a ></a><img src="Linkedinicon.jpg" alt=" " class="linkedin"></a></li>
+      <li><a ><img src="angellist.jpg" alt=" " class="angellist"></a></li>
+     <li><a ><img src="twitter.jpg" alt=" " class="twitter"></a></li>
+      <li><a ><img src="medium.jpg" alt=" " class="medium"></a></li>
+    </ul>
+</div>
+
+
```
Comment (rloterh) [inline project1.html]: - [ ] OPTIONAL
Kindly ensure not to add leave empty lines in your code which do not serve any purpose, so as to keep your code cleaner.

File: project1.html [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+   <div class="one">LET'S CONNECT </div>
+
+<div>
+  <ul>
+     <li><a ><img src="github.jpg" alt=" " class="github"></a></li>
+    <li><a ></a><img src="Linkedinicon.jpg" alt=" " class="linkedin"></a></li>
+      <li><a ><img src="angellist.jpg" alt=" " class="angellist"></a></li>
+     <li><a ><img src="twitter.jpg" alt=" " class="twitter"></a></li>
+      <li><a ><img src="medium.jpg" alt=" " class="medium"></a></li>
+    </ul>
```
Comment (rloterh) [inline project1.html]: - [ ] OPTIONAL
Your code is **not properly indented** and therefore your code formatting is inconsistent. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Senan,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Design matches provided template

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**